src/data-workspace/section-form/section.js

```diff
@@ -88,44 +106,61 @@ export function SectionFormSection({ section, dataSetId, globalFilterText }) {
                             className={styles.headerCell}
                         >
                             <div className={styles.labelWrapper}>
-                                <div className={styles.title}>
-                                    {section.displayName}
+                                <div
+                                    className={styles.collapseIcon}
+                                    tabIndex={collapsible ? 0 : -1}
+                                    onClick={onSectionHeadClicked}
+                                    onKeyDown={onSectionHeadEntered}
+                                >
+                                    {collapsible &&
+                                        (showSectionContent ? (
+                                            <IconChevronUp16 color="white" />
```

Comment:
we also have a constant value for white, in case we're worried that our design system is ever going to change DHIS2 white 😅 https://github.com/dhis2/ui/blob/master/constants/src/colors.js#L74